docs/refs.bib

```diff
@@ -0,0 +1,51 @@
+Mitigations.rst
+
+Agarwal, Beygelzimer, Dudik, Langford, Wallach “A Reductions Approach to Fair Classification”, ICML, 2018.
+@article{agarwal2018reductions,
```

Comment:
I think we should be using something more similar to this: https://dblp.uni-trier.de/rec/conf/icml/AgarwalBD0W18.html?view=bibtex&param=0
(so `@inproceedings` rather than `@article`)

Reply:
Got it I will change to `@inproceedings`, do you mean to use the `condensed` format? It does not have URLs so the references will not link back to the papers.

Reply:
For reference, the citations will look like this:

**Standard**
<img src="https://user-images.githubusercontent.com/50991099/156824881-f2dc647d-71d6-4ea9-89cf-4d6ea54a7516.png" height=300>

**Condensed**
<img src="https://user-images.githubusercontent.com/50991099/156825413-87b91a8f-8b4c-4e8f-a531-2e0567544142.png" height=250>

Reply:
I think that having a URL is definitely nice to have, but the full standard citation seems very verbose. I would go for something in between, maybe (also fixing my name, which is getting a bit mangled):
```
@inproceedings{agarwal2018reductions,
  author    = {Alekh Agarwal and
               Alina Beygelzimer and
               Miroslav Dudík and
               John Langford and
               Hanna M. Wallach},
  title     = {A Reductions Approach to Fair Classification},
  booktitle = {Proceedings of the 35th International Conference on Machine Learning,
               {ICML} 2018},
  pages     = {60--69},
  year      = {2018},
  url       = {http://proceedings.mlr.press/v80/agarwal18a.html},
  biburl    = {https://dblp.org/rec/conf/icml/AgarwalBD0W18.bib},
  bibsource = {dblp computer science bibliography, https://dblp.org}
}
```

Reply:
FWIW I'd rather add more info than less. It's at the bottom of the page anyway...

Reply:
> (also fixing my name, which is getting a bit mangled

Sorry about that, I'll change it.

I think let's go with the condensed version, we will add in the `url` field to it. The verbosity mainly comes from the `booktitle`.

It does not look too verbose, nor too less -
![image](https://user-images.githubusercontent.com/50991099/156881188-1f0839db-2720-4065-af7f-f8ce92c9a2b5.png)